### PR TITLE
More consistent api endpoints for adding users to teams and orgs

### PR DESCRIPTION
**Pull request by justinvdm**
At the moment we do:

```http
POST /teams/23/users/
{"user_id": 21}
```

It would be more consistent with the rest of the api to be able to do:

```http
PUT /teams/23/users/21
```

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler ready for review

File: docs/http_api.rst [hunk 1]
```diff
@@ -585,7 +581,7 @@ to belong to exactly one organization, but an organization can have many teams.
 
     See `Remove permission from team`_. Limited to teams that belong to the organization.
 
-.. http:post:: /organizations/(int:organization_id)/teams/(int:team:id)/users/
+.. http:post:: /organizations/(int:organization_id)/teams/(int:team:id)/users/(int:user_id)/
```
Comment (rudigiesler) [inline docs/http_api.rst]: Should this not also be `put`?

**Comment by rudigiesler** [comment]
Just the one comment from me, looks good otherwise

**Comment by justinvdm** [comment]
@rudigiesler Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**